docs/examples/agrivoltaics/plot_diffuse_PAR_Spitters_relationship.py

```diff
@@ -0,0 +1,145 @@
+"""
+Calculating the diffuse PAR using Spitter's relationship
+=========================================================
+
+This example demonstrates how to calculate the diffuse photosynthetically
+active radiation (PAR) from total PAR using Spitter's relationship.
+"""
+
+# %%
+# The photosynthetically active radiation (PAR) is a key component in the
+# photosynthesis process of plants. As in photovoltaic systems, PAR can be
+# divided into direct and diffuse components. The diffuse fraction of PAR
+# with respect to the total PAR is important in agrivoltaic systems, where
+# crops are grown under solar panels. The diffuse fraction of PAR can be
+# calculated using the Spitter's relationship [1]_ implemented in
+# :py:func:`~pvlib.irradiance.diffuse_par_spitters`.
+# This model requires the solar zenith angle and the fraction of the broadband
+# radiation that is diffuse as inputs.
```

Comment:
```suggestion
# irradiance that is diffuse as inputs.
```

Reply:
To (perhaps) head off a discussion of whether "radiation" or "irradiance" _should_ be used here: radiation is usually used when referring to the source, and irradiance (or irradiation) when referring to the receiver. So the sun emits radiation, and a tilted plane on the earth's surface receives solar irradiation. Either perspective seems OK to me in the context of this PR. But mixing PAR (..."radiation") with "irradiance" may be confusing to some.